project/src/main/webapp/upload-page/upload.html

```diff
@@ -5,25 +5,103 @@
     <title>Report a Case</title>
     <link rel="stylesheet" href="upload_style.css">
     <script src="upload_script.js"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+
   </head>
   <body>
       <div id="content"> 
-        <form action="index.html">
+        <form action="../index.html">
             <input type="submit" value="Return to Main Page" />
         </form>
-        <center>
-            <h1>Upload a report!</h1>
-            <form action="/report-handler" method="POST" enctype="multipart/form-data">
-            <!-- <form action="/report-upload" method="POST" enctype="multipart/form-data"></form> -->
-                <p>Please upload a quick description of the collision:</p>
-                <textarea name="text-input">E.g. date, time-of-day, car types of those involved, etc.</textarea>
+        <center><h1>Upload a report!</h1></center>
+<<<<<<< HEAD
```

Comment:
Please resolve this diff by selecting the HTML sections that you want to keep. The section between the <<< HEAD and === markers are your local changes, and the section between === and >>>> ec2c0ce4d528... is how this same section looks in the commit your merging.